notebooks/examples/quickstart_in_5_minutes.ipynb

```diff
@@ -0,0 +1,2112 @@
+{
```

Comment:
<p>all this 4 blocks of code preparations are mostly noise, if possible I'd put them in a single cell, even a collapsed one if possible, as I don't think most users will care to see this. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/237/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>